src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -3,11 +3,25 @@
 {# Widgets #}
 
 {% block money_widget -%}
-    {% if not valid %}
-        {% set group_class = ' form-control is-invalid' %}
-        {% set valid = true %}
-    {% endif %}
-    {{- parent() -}}
+    {%- set prepend = not (money_pattern starts with '{{') -%}
+    {%- set append = not (money_pattern ends with '}}') -%}
+    {%- if prepend or append -%}
+        <div class="input-group{{ group_class|default('') }}">
```

Comment:
is `group_class` defined here ?

Reply:
Not in this block no. The abstract form has the same code: https://github.com/symfony/symfony/blob/v4.0.4/src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_base_layout.html.twig#L14

I guess it was just copied and added some modifications.